### ISSUE TITLE
Z1. Remove gcis:hasTemporalExtents and gcis:hasSpatialExtents, replace with usage of dublin core terms properties

### ISSUE THREAD

**Issue by zednis** [USGCRP/gcis-ontology#11]
| Change Proposal | Z1 |
| :-- | :-: |
| Point of Contact | @zednis |
| Date | 2015-05-26 |
## Proposal

Remove properties **gcis:hasTemporalExtents** and **gcis:hasSpatialExtents** from the ontology.
## Background

**gcis:hasTemporalExtents** and **gcis:hasSpatialExtents** are defined in the ontology as way to state the temporal and spatial extents of a model run.

```
gcis:hasSpatialExtents a owl:ObjectProperty ;
    rdfs:label "Has Spatial Extents" ;
    rdfs:comment "An entity of model run may have spatial extents." ;
    rdfs:domain gcis:ModelRun ;
    rdfs:range gcis:SpatialExtents .

gcis:hasTemporalExtents a owl:ObjectProperty ;
    rdfs:label "Has Temporal Extents" ;
    rdfs:comment "An entity of model run may have temporal extents." ;
    rdfs:domain gcis:ModelRun ;
    rdfs:range gcis:TemporalExtents .
```

These properties have broad labels but narrow usage since they are confined to model runs.  They also seem redundant with the [dcterms:temporal](http://dublincore.org/documents/dcmi-terms/#terms-temporal) and [dcterms:spatial](http://dublincore.org/documents/dcmi-terms/#terms-spatial) properties.

I believe we should drop usage of **gcis:hasSpatialExtents** and **gcis:hasTemporalExtents** and use **dcterms:spatial** and **dcterms:temporal** in their stead.
## Does this change affect the semantics of the ontology?

YES

Removes a locally defined property which will be replaced in instance data with a property from dublin core.
## Describe in detail the change being requested
1. Delete **gcis:hasTemporalExtents** and **gcis:hasSpatialExtents**
2. Use **dcterms:spatial** and **dcterms:temporal** in the instance data
## Describe the reason for the change
1. Remove GCIS-defined properties where dublin core properties could be used.  
2. Replacement properties have broader domains which will better meet our needs.
## What are the anticipated impacts
- use of **dcterms:spatial** and **dcterms:temporal** for spatial and temporal extents in the instance data
- new inferences from the dcterms property range
- individuals with spatial and temporal extents will no longer be inferred to have type **gcis:ModelRun**
## What are the risks of not performing the change?
- inferences from current ontology incorrectly assign type **gcis:ModelRun** to non-model individuals given a temporal or spatial extent.
## What is the expected technical effort to support the change?

Expected effort to support the change is low.
- remove 2 properties from the ontology
- update instance data scripts to use **dcterms:spatial** and **dcterms:temporal**
- update any queries or processed that used the removed properties
## Describe alternatives considered

We could keep the existing properties but broaden (or remove) their domains.
## Related change proposals

N/A
## Notes

N/A

**Comment by bduggan** [comment]
Sounds fine to me. :+1:

**Comment by zednis** [comment]
I have updated this request change to suggest using dct:temporal and dct:spatial instead of gcis:hasTemporalExtents and gcis:hasSpatialExtents, respectively.

**Comment by justgo129** [comment]
@zednis, I see that dcterms:spatial (http://dublincore.org/documents/dcmi-terms/#terms-spatial) has range:Location.  dcterms:Location has definition: "A spatial region or named place."  Does the delineation of various coordinates (for dcterms:Spatial) satisfy "a spatial region?"  I've been unable to locate a more specific meaning for "spatial region."

**Comment by zednis** [comment]
@justgo129 are you asking if a bounding box would qualify as a spatial region?  Yes.

DC Terms does not provide a set of properties or attributes we must use to describe a location.  We can use any other vocabulary and attributes/properties as make sense to describe a location in our context.

There are some good examples from the User Guide/Publishing Metadata for Dublin Core in RDF (always a good source for looking up the 'approved' way of using dublin core in RDF)

from http://wiki.dublincore.org/index.php/User_Guide/Publishing_Metadata#dcterms:spatial

example 1 - a spatial region with coordinates

```
ex:myData dcterms:title "Analysis of rocks collected in Perth" ;
          dcterms:spatial _:thisPlace .

_:thisPlace ex:east "115.85717" ;
            ex:north "-31.95301" ;
```

example 2 - a general region by description (no coordinates or spatial information)

```
ex:myData dcterms:title "The growth of trees in the suptropical highlands" ;
          dcterms:spatial _:Cwb .

_:Cwb rdfs:label "Cwb"
      dc:source "Köppen-Geiger Climate Classification" ;
      ex:mainClimates "warm temperate" ;
      ex:precipitation "winter dry" ;
      ex:temperature "warmest month averaging below 22°C" .
```

We can describe a spatial region or location in whatever way makes sense for our needs.

**Comment by zednis** [comment]
Really our only requirement is that the Location be a RDF resource and make sense semantically as some kind of location.

**Comment by justgo129** [comment]
Sure thing.  Yes, I'm inquiring about a bounding box, polygon, any vector-form geometric representation.  I'm happy to see this flexibility employed, and did not previously know about the wiki (thanks!).  I accept the suggested change.

**Comment by zednis** [comment]
Closed with #13
